gluster_exporter/metric_brick.go

```diff
@@ -0,0 +1,187 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"syscall"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	brick_labels = []string{
+		"host",
+		"id",
+		"brick_path",
+		"volume",
+		"subvolume",
+	}
+
+	glusterBrickUsedCapacity = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "gluster",
+			Name:      "brick_used_capacity",
+			Help:      "Used capacity of gluster bricks",
+		},
+		brick_labels,
+	)
+
+	glusterBrickFreeCapacity = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "gluster",
+			Name:      "brick_free_capacity",
+			Help:      "Free capacity of gluster bricks",
+		},
+		brick_labels,
+	)
+
+	glusterBrickAllCapacity = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "gluster",
+			Name:      "brick_all_capacity",
+			Help:      "Total capacity of gluster bricks",
+		},
+		brick_labels,
+	)
+
+	glusterBrickDiskInodesAll = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "gluster",
+			Name:      "brick_disk_inodes_all",
```

Comment:
`_disk_` can be removed, similar to other metric names. Also `all` can be replaced with `total`

    brick_inodes_total
    brick_inodes_free
    brick_inodes_used

Reply:
Ack